### ISSUE TITLE
[FEATURE REQUEST] Weekend days indicated with other color

### ISSUE THREAD

**Issue by hastaboera** [Blueforcer/awtrix3#547]
# Feature Request

### Is your feature request related to a problem?

No

### Describe the solution / feature you'd like

Add a different color to the weekend days (sat & sun) indicated on the Time app. 

### Describe alternatives you've considered

/

### Additional context

Took me a while to figure out it are the weekdays indicated as the dots. Having the weekend colored gives a quicker glance of what day of the week it is.

**Comment by wolfspirituk** [comment]
Just an alternative implementation .... 
I wanted this as well - so I used an automation in Home Assistant to reset the colour with WDCA in the settings at the beginning of each day. Pick your own colours. I guess anything that can send the MQTT message or the the POST  can do the same thing.

**Comment by regislutter** [comment]
I was going to request the same feature :)

**Comment by regislutter** [comment]
> Just an alternative implementation .... I wanted this as well - so I used an automation in Home Assistant to reset the colour with WDCA in the settings at the beginning of each day. Pick your own colours. I guess anything that can send the MQTT message or the the POST can do the same thing.

This would only make the color of the active weekend day different. I think, like me, it's more like having the 5 days of the week in white/50%, and the weekend days in green/50% for example. Then, the active day would be white/100% or green/100% depending if it's a weekend day or not.

**Comment by a-bianucci** [comment]
> Just an alternative implementation .... I wanted this as well - so I used an automation in Home Assistant to reset the colour with WDCA in the settings at the beginning of each day. Pick your own colours. I guess anything that can send the MQTT message or the the POST can do the same thing.

How did you do this?  I have been even trying to change the color of the time with TIM_COL and I can't get that to work either.

**Comment by wolfspirituk** [comment]
> > Just an alternative implementation .... I wanted this as well - so I used an automation in Home Assistant to reset the colour with WDCA in the settings at the beginning of each day. Pick your own colours. I guess anything that can send the MQTT message or the the POST can do the same thing.
> 
> How did you do this? I have been even trying to change the color of the time with TIM_COL and I can't get that to work either.

I use this bit of HA YAML and call it just after midnight. There are other things in here so just look at the CHCOL WDCI and WDCA settings. This sets up a payload that I send via MQTT. You can do it in other ways - the code and data are the important bit. Works a treat.
  ma_reset_settings:
    alias: MA - Reset Settings
    fields:
      request_on:
        description: ''
    sequence:
      - variables:
          return_value:
            the_topic_root: settings
            the_payload: |-
              {
                {%- if now().weekday() >= 5 %}
                  "CHCOL": [ 0, 255, 0 ],
                  "WDCA": [ 0, 255, 0 ],
                {%- elif states('binary_sensor.workday_sensor') == 'on' %}
                  "CHCOL": [ 255, 0, 0 ],
                  "WDCA": [ 255, 0, 0 ],
                {%- else %}
                  "CHCOL": [ 0, 0, 255 ],
                  "WDCA": [ 0, 0, 255 ],
                {%- endif %}
                {%- if request_on %}
                  "BRI": 200,
                  "ABRI": false,
                {%- else %}
                  "BRI": 7,
                  "ABRI": false,
                {%- endif %}
                "WDCI": [ 64, 64, 64 ],
                "TEFF": 10,
                "BLOCKN": true,
                "TSPEED": 333,
                "UPPERCASE": false,
                "VOL": 15
              }
      - stop: Finished
        response_variable: return_value
    description: ""

**Comment by HaraldGithub** [comment]
I would support this feature request! :-)

**Comment by ray154856235** [comment]
I would support this feature request! :-)  
but not only weekend days also with calendar

**Comment by fredbcode** [comment]
Yes great idea, I like to see the active day in a différent color
